docs/doxygen/migration/migration.txt

```diff
@@ -0,0 +1,7 @@
+/**
+@page migration_guide Migration Guide
+@brief Migration guide for v3 libraries to the 202009.00 release.
```

Comment:
```suggestion
@brief Migration guide for v3 library to the 202009.00 release.
```

Reply:
plz check [the commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1226/commits/f504c65a1d8030ae4237312b7b63b563b8700ca8).